src/pages/register.tsx

```diff
@@ -0,0 +1,211 @@
+                        อาหารที่แพ้ (ไม่มีให้ใส่ -)
+                    </label>
+                    <input
+                        type="text"
+                        name="foodAllergy"
+                        id="foodAllergy"
+                        className={inputStyle}
+                        required
+                    />
+                    <label htmlFor="" className="mb-1">
+                        ข้อจำกัดทางอาหาร (ไม่มีให้ใส่ -)
+                    </label>
+                    <input
+                        type="text"
+                        name="foodRestriction"
+                        id="foodRestriction"
+                        className={inputStyle}
+                        required
+                    />
+                    <hr className="w-full h-1 mt-3 mb-5" />
+                    <label htmlFor="" className="mb-1">
+                        เบอร์ติดต่อฉุกเฉิน
+                    </label>
+                    <input
+                        type="text"
+                        name="emergencyNo"
+                        id="emergencyNo"
+                        className={inputStyle}
+                        required
+                    />
+                    <label htmlFor="" className="mb-1">
+                        ความสัมพันธ์กับตนเอง
+                    </label>
+                    <input
+                        type="text"
+                        name="emergencyRel"
+                        id="emergencyRel"
+                        className={inputStyle}
+                        required
+                    />
+                </div>
+                <div className="w-10/12 flex flex-row justify-start items-start mt-8 mb-2">
+                    <input
+                        type="checkbox"
+                        name="flask"
+                        id="flask"
+                        className=" bg-input-gray border-none w-7 h-7 mr-3"
+                    />
+                    <label htmlFor="flask" className="text-left w-3/4">
+                        รับกระติกน้ำ Zero waste ที่โต๊ะลงทะเบียนในวันที่ 5
+                    </label>
+                </div>
+                <div className="w-10/12 flex flex-row justify-start items-start mt-3 mb-5">
+                    <input
+                        type="checkbox"
+                        name="tc"
+                        id="tc"
+                        className=" bg-input-gray border-none w-7 h-7 mr-3"
+                    />
+                    <label htmlFor="tc" className="text-left ">
+                        ยอมรับ{' '}
+                        <Link href={''} className=" underline">
+                            ข้อตกลงและเงื่อนไขการใช้งาน
+                        </Link>
+                    </label>
+                </div>
+                <button
+                    type="submit"
+                    className="text-xl text-white font-bold bg-[#E95682] rounded-full py-2 px-14 my-10 mb-16 shadow-regisSubmit"
```

Comment:
The `tailwind.config.js file` already has the color **pink** configured. Please use it instead of an arbitrary value.